lib/generator/implementation.rb

```diff
@@ -1,5 +1,6 @@
 require 'delegate'
 require 'forwardable'
+require_relative 'template_values'
```

Comment:
Any reason you've used `require_relative` here?  and `relative` everywhere else?